images/golang-dind/build.yaml

```diff
@@ -1,6 +1,12 @@
 name: golang-dind # Name of the image to be built
 
+variants:
+  "1.13.4":
+    arguments:
+      BASE_IMAGE: "${_REGISTRY}/bazelbuild:latest-1.0.0"
```

Comment:
Probably best to pin to a specific version here, else it'll be difficult to know exactly what base image it ended out being built with.

Reply:
e.g. eu.gcr.io/jetstack-build-infra-images/bazelbuild:20191016-eff358a-1.0.0